nixos/modules/services/web-apps/stirling-pdf.nix

```diff
@@ -54,7 +75,7 @@ in
           # See https://github.com/Stirling-Tools/Stirling-PDF/blob/main/src/main/java/stirling/software/SPDF/config/ExternalAppDepConfig.java#L42
           which
           unpaper
-          libreoffice
+          (libreoffice.override { dbusVerify = false; })
```

Comment:
Please add a comment above this so that other people looking at the module know why it's overridden.